CONTRIBUTING.md

```diff
@@ -1,21 +0,0 @@
-# Contributing Guide
```

Comment:
Why was this removed? We can keep these instructions for contributing to the docs and extend it in future for other parts?

Reply:
Readme wont contain anything else. So, I moved all of contributing.md to readme